web/src/buddy_data.js

```diff
@@ -66,7 +68,26 @@ export function level(user_id) {
     }
 }
 
-export function compare_function(a, b) {
+export function user_matches_narrow(user_id, current_sub, pm_ids) {
+    if (current_sub) {
+        return stream_data.is_user_subscribed(current_sub.stream_id, user_id);
+    }
```

Comment:
Would a `stream_id` be a better parameter for this function than `current_sub`?

Reply:
Sure! Changed that.